homeassistant/components/tado/repairs.py

```diff
@@ -17,11 +17,10 @@ def manage_water_heater_fallback_issue(
     integration_overlay_fallback: str | None,
 ) -> None:
     """Notify users about water heater respecting fallback setting."""
-    if (
-        integration_overlay_fallback
-        in [CONST_OVERLAY_TADO_DEFAULT, CONST_OVERLAY_MANUAL]
-        and len(water_heater_entities) > 0
-    ):
+    if integration_overlay_fallback in [
+        CONST_OVERLAY_TADO_DEFAULT,
+        CONST_OVERLAY_MANUAL,
+    ]:
```

Comment:
Seeing how this list is read-only, you could make it a tuple - will use less memeory:
```python
if integration_overlay_fallback in (
    CONST_OVERLAY_TADO_DEFAULT,
    CONST_OVERLAY_MANUAL,
):
    ```